### ISSUE TITLE
Quote Loader error on all securities

### ISSUE THREAD

**Issue by rsworden59** [mrbray99/moneydanceproduction#1]
Maybe the Moneydance 2024 update broke this? I updated MD to 2024 (5117) on 2024-04-16. On 2024-04-19 I noticed QL failing. After Get Prices it says "0 Prices Loaded 9 errors". I tried reinstalling the extension but still failing.

**Comment by rsworden59** [comment]
[Quote Loader errors.txt](https://github.com/mrbray99/moneydanceproduction/files/15049189/Quote.Loader.errors.txt)

**Comment by trayhumphrey** [comment]
Yes, I can report same as rsworden59. No prices loaded, just errors.

**Comment by evmazzone** [comment]
I haven't updated and am still on 2023.3 and am getting nothing but errors.  This is disturbing.

**Comment by mrbray99** [comment]
Please use the public discussions to report issues. There is information there about QL errors

**Comment by rsworden59** [comment]
Can you please direct me to where that is? I did not find anything, so I went to Github Issues.

Thanks.
